cypress/support/commands/common.js

```diff
@@ -657,46 +668,73 @@ Cypress.Commands.add('c_closeNotificationHeader', () => {
 })
 
 Cypress.Commands.add('c_skipPasskeysV2', (options = {}) => {
-  const {
-    language = 'english',
-    retryCount = 0,
-    maxRetries = 3,
-    withoutContent = false,
-  } = options
-  if (withoutContent == true) {
-    cy.get('.effortless-login-modal')
-      .should(() => {})
-      .then(($el) => {
-        if ($el.length) {
-          cy.log('Skipped Passkeys prompt !!!')
-          cy.get('.effortless-login-modal__header').click()
-        } else if (retryCount < maxRetries) {
-          cy.wait(300)
-          cy.log(
-            `Passkeys prompt did not appear, Retrying... Attempt ${retryCount + 1}`
-          )
-          cy.c_skipPasskeysV2({ ...options, retryCount: retryCount + 1 })
-        }
-      })
-  } else {
-    cy.fixture('common/common.json').then((langData) => {
-      const lang = langData[language]
-      cy.findByText(lang.passkeysModal.title)
-        .should(() => {})
-        .then(($el) => {
-          if ($el.length) {
-            cy.findByText(lang.passkeysModal.maybeLaterBtn).click()
-            cy.log('Skipped Passkeys prompt !!!')
-          } else if (retryCount < maxRetries) {
-            cy.wait(300)
-            cy.log(
-              `Passkeys prompt did not appear, Retrying... Attempt ${retryCount + 1}`
-            )
-            cy.c_skipPasskeysV2({ ...options, retryCount: retryCount + 1 })
-          }
-        })
-    })
-  }
+  cy.log('should have no passkey')
+  // const {
+  //   language = 'english',
+  //   retryCount = 0,
+  //   maxRetries = 3,
+  //   withoutContent = false,
+  // } = options
+  // if (withoutContent == true) {
+  //   cy.get('.effortless-login-modal')
+  //     .should(() => {})
+  //     .then(($el) => {
+  //       if ($el.length) {
+  //         cy.log('Skipped Passkeys prompt !!!')
+  //         cy.get('.effortless-login-modal__header').click()
+  //       } else if (retryCount < maxRetries) {
+  //         cy.wait(300)
+  //         cy.log(
+  //           `Passkeys prompt did not appear, Retrying... Attempt ${retryCount + 1}`
+  //         )
+  //         cy.c_skipPasskeysV2({ ...options, retryCount: retryCount + 1 })
+  //       }
+  //     })
+  // } else {
+  //   cy.fixture('common/common.json').then((langData) => {
+  //     const lang = langData[language]
+  //     cy.findByText(lang.passkeysModal.title)
+  //       .should(() => {})
+  //       .then(($el) => {
+  //         if ($el.length) {
+  //           cy.findByText(lang.passkeysModal.maybeLaterBtn).click()
+  //           cy.log('Skipped Passkeys prompt !!!')
+  //         } else if (retryCount < maxRetries) {
+  //           cy.wait(300)
+  //           cy.log(
+  //             `Passkeys prompt did not appear, Retrying... Attempt ${retryCount + 1}`
+  //           )
+  //           cy.c_skipPasskeysV2({ ...options, retryCount: retryCount + 1 })
+  //         }
+  //       })
+  //   })
+  // }
+})
+
+Cypress.Commands.add('disablePasskey', (options = {}) => {
```

Comment:
To align with current naming convention:
```suggestion
Cypress.Commands.add('c_disablePasskey', (options = {}) => {
```